ebs-recover-and-replace/src/ebs_recover_and_replace/__init__.py

```diff
@@ -10,9 +10,45 @@
 from collections import Counter
 
 
-def separator(width=72):
+class colours:
+    dblue = "\033[0;34m"
+    dred = "\033[0;31m"
+    byellow = "\033[1;33m"
+    bgreen = "\033[1;32m"
+    bold = "\033[1m"
+    end = "\033[0m"
+
+
+def format_output(message="", style="info", indent=4):
+    prefix = ""
+    suffix = ""
+    if style == "error":
+        print(f"{colours.dred}{style.capitalize()}:{colours.end} {message}")
+        sys.exit(1)
```

Comment:
I don't think we should be exiting from a `format_output` function. I think the onus should be on the calling code to dictate when to exit

Reply:
Updated in [a541ddb](https://github.com/companieshouse/platform-tools-python/pull/4/commits/a541ddb4a3187b0ba61bfe245612bac8d46e3957)